packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -218,6 +226,18 @@ export default class QuanticResultQuickview extends LightningElement {
     });
   }
 
+  handleLoadingStateChange() {
+    this._isLoading = false;
+  }
```

Comment:
Just to play it safe it would be better to stop the propagation of the event after handling it.
```suggestion
  handleLoadingStateChange(event) {
    event.stopPropagation();
    this._isLoading = false;
  }
```